config.template

```diff
@@ -1,88 +1,109 @@
 [case]
-# ACME case configuration
-casename = 20160805v0atm.A_WCYCL1850_v0atm.ne30_oEC.edison.alpha7_00
-native_res = ne30
-short_term_archive = 0
-ref_casename_v0 = B1850C5_ne30_v0.4
+
+# casename is a name that identifies the simulation being analyzed.
+casename = casename
+# ref_casename is the name of a reference run to compare agains (or None
+# to turn off comparison with a reference, e.g. if no reference case is
+# available)
+ref_casename = None
 
 [input]
-# names of namelist and streams files
+
+# directory containing model results
+basedir = /dir/to/model/output
```

Comment:
@milenaveneziani, I think maybe this issue is too big to address in this PR and should be make a separate issue.

Again, I feel strongly that any solution shouldn't be aware of the concept that are ACME-specific such as "short-term archiving".  Instead, we need to have a flexibility in finding the appropriate files (both the data sets themselves and their corresponding namelist and streams files).  I have a feeling this could be logistically challenging because files may not be in the locations specified relative to the streams files after archiving (unless the namelist and streams files are archived along with the data, which would be *very* helpful).